ivy/data_classes/array/array.py

```diff
@@ -144,6 +144,10 @@ def _init(self, data, dynamic_backend=None):
             self._data = data
         elif isinstance(data, np.ndarray):
             self._data = ivy.asarray(data)._data
+        elif ivy.experimental.is_ivy_sparse_array(data):
```

Comment:
let's use `ivy.is_ivy_sparse_array` instead, same applies to the next `if`

Reply:
updated 👍🏼